scipy/datasets/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""
+================================
+Datasets (:mod:`scipy.datasets`)
+================================
+
+.. currentmodule:: scipy.datasets
+
+.. autosummary::
+   :toctree: generated/
+
+   ascent - Get example image for processing
+   face - Get example image for processing
+   electrocardiogram - Load an example of a one-dimensional signal.
+
+
+Usage of Datasets
```

Comment:
This isn't quite usage, more like a "how does it work under the hood". How about an actual `Usage` section above, which only explains that you call `dataset_name()` and it returns you the ndarray, dict or other Python data structure, and that it is cached so downloads over the network once? And then below that this `How dataset retrieval and storage works` section.

Reply:
I've tried to improve this, how does it look now?

Reply:
Looks better now. I pushed one commit with small tweaks like using the correct reST header underlines and removing the question mark from the last section title.